website/pages/en/deploying/hosted-service.mdx

```diff
@@ -46,6 +46,19 @@ graph init --from-example --product hosted-service <GITHUB_USER>/<SUBGRAPH_NAME>
 
 The example subgraph is based on the Gravity contract by Dani Grant that manages user avatars and emits `NewGravatar` or `UpdateGravatar` events whenever avatars are created or updated. The subgraph handles these events by writing `Gravatar` entities to the Graph Node store and ensuring these are updated according to the events. Continue on to the [subgraph manifest](/developing/creating-a-subgraph#the-subgraph-manifest) to better understand which events from your smart contracts to pay attention to, mappings, and more.
 
-## Supported Networks on the hosted service
+
+### From a Proxy Contract
+
+To build a subgraph tailored for monitoring a Proxy contract, initialize the subgraph by specifying the address of the implementation contract. Once the initialization process is concluded, the last step involves updating the network name in the subgraph.yaml file to the address of the Proxy contract.
+You can use the command below.
```

Comment:
I think this should either be part of the previous line, or there should be a line between the two if it's meant to be a separate paragraph.

Reply:
Okay, I'll fix that.